website/blog/2023-12-01-AutoGenStudio/index.mdx

```diff
@@ -189,6 +191,21 @@ As we continue to develop and refine AutoGen Studio, the road map below outlines
 - **Expansion of Agent Skills**: We will work towards improving the workflow for authoring, composing and reusing agent skills.
 - **Community Features**: Facilitation of sharing and collaboration within AutoGen Studio user community is a key goal. We're exploring options for sharing sessions and results more easily among users and contributing to a shared repository of skills, agents, and agent workflows.
 
+
+
+## Contribution Guide
+
+We welcome contributions to AutoGen Studio. We recommend the following general steps to contribute to the project:
+
+- Review the overall AutoGen project [contribution guide](https://github.com/microsoft/autogen?tab=readme-ov-file#contributing)
+- Please review the AutoGen Studio [roadmap](https://github.com/microsoft/autogen/issues/737) to get a sense of the current priorities for the project. Help is appreciated especially with Studio issues tagged with `help-wanted`
+- Please initiate a discussion on the roadmap issue or a new issue to discuss your proposed contribution.
+- Please review the autogenstudio dev branch here [dev branch](https://github.com/microsoft/autogen/tree/autogenstudio) and use as a base for your contribution. This way, your contribution will be aligned with the latest changes in the AutoGen Studio project.
+- Submit a pull request with your contribution!
+- If you are modifying AutoGen Studio, it has its own devcontainer. See instructions in `.devcontainer/README.md` to use it
+- Please use the tag `studio` for any issues, questions, and PRs related to Studio

```

Comment:
```suggestion
- Review the overall AutoGen project [contribution guide](https://github.com/microsoft/autogen?tab=readme-ov-file#contributing).
- Please review the AutoGen Studio [roadmap](https://github.com/microsoft/autogen/issues/737) to get a sense of the current priorities for the project. Help is appreciated especially with Studio issues tagged with `help-wanted`.
- Please initiate a discussion on the roadmap issue or a new issue to discuss your proposed contribution.
- Please review the autogenstudio dev branch here [dev branch].(https://github.com/microsoft/autogen/tree/autogenstudio) and use as a base for your contribution. This way, your contribution will be aligned with the latest changes in the AutoGen Studio project.
- Submit a pull request with your contribution!
- If you are modifying AutoGen Studio in vscode, it has its own devcontainer. See instructions in `.devcontainer/README.md` to use it.
- Please use the tag `studio` for any issues, questions, and PRs related to Studio.
```

Reply:
If you add periods here add it everywhere. Including README. I personally find periods for enumerating large number of short bullets unnecessary but consistency is more important than anything else.